openlibrary/templates/lib/nav_head.html

```diff
@@ -103,18 +113,19 @@
         # dev environment), this get_internet_archive_id returns None. Setting it to an
         # empty string returns a valid image ¯\_(ツ)_/¯
         'image': 'https://archive.org/services/img/' + (get_internet_archive_id(ctx.user.key) or ''),
-        'links': [
-          { "href": ctx.user.key, "text": _("My Profile") },
-          { "href": homepath() + "/account/loans", "text": _("My Loans") },
-          { "href": ctx.user.key + "/lists", "text": _("My Lists") },
-          { "href": "/account/books", "text": _("My Reading Log") },
-          { "href": "/account/books/already-read/stats", "text": _("My Reading Stats") },
-          { "href": homepath() + "/account", "text": _("Settings") },
-          { "post": "/account/logout", "text": _("Log out") }
-        ]
+        'links': loginLinks
       }
     $# Add book notes link between 'My Reading Stats' and 'Settings' for beta testers:
     $if ctx.user.is_beta_tester():
       $accountProps['links'].insert(5, { "href": "/account/books/notes", "text": _("My Book Notes") })
     $:render_template("lib/header_dropdown", accountProps )
+
+  $:render_template("lib/header_dropdown", hamburgerProps )
+
 </header>
+<div class="header-bar mobile">
```

Comment:
And closing tag.
```suggestion
<header class="header-bar mobile">
```

Reply:
I haven't done this because changing it to header, would trigger 
```
// For pages without a header we don't want a margin.
// This might be better placed in the header component in future
// but is kept here for backwards compatibility with v1 pages
header+#test-body-mobile {
    margin-top: 20px;
}
```

Reply:
Done